src/yaml/yaml-network-parser.c

```diff
@@ -260,23 +296,27 @@ static int parse_yaml_node(YAMLManager *m, yaml_document_t *dp, yaml_node_t *nod
                 for (i = node->data.sequence.items.start; i < node->data.sequence.items.top; i++) {
                         n = yaml_document_get_node(dp, *i);
                         if (n)
-                                (void) parse_yaml_node(m, dp, n, network);
+                                (void) parse_yaml_node(m, dp, n, network, link);
                 }
         }
                 break;
         case YAML_MAPPING_NODE:
-                (void) parse_network_config(m, dp, node, network);
-                (void) parse_wifi_access_points_config(m, dp, node, network);
-                (void) parse_route_config(m, dp, node, network);
-
                 for (p = node->data.mapping.pairs.start; p < node->data.mapping.pairs.top; p++) {
                         n = yaml_document_get_node(dp, p->key);
-                        if (n)
-                                (void) parse_yaml_node(m, dp, n, network);
-
-                        n = yaml_document_get_node(dp, p->value);
-                        if (n)
-                                (void) parse_yaml_node(m, dp, n, network);
+                        if (string_equal(scalar(n), "match") || string_equal(scalar(n), "link")) {
```

Comment:
`match` is for https://www.freedesktop.org/software/systemd/man/systemd.link.html#%5BMatch%5D%20Section%20Options. Only ```link:``` is useful

Reply:
done